cmd/tink-worker/worker.go

```diff
@@ -74,19 +82,30 @@ func processWorkflowActions(client pb.WorkflowSvcClient) error {
 				switch wfContext.GetCurrentActionState() {
 				case pb.ActionState_ACTION_SUCCESS:
 					if isLastAction(wfContext, actions) {
-						log.Infof("Workflow %s completed successfully\n", wfID)
+						ll := l.With("workflowStatus", pb.ActionState_ACTION_SUCCESS.String())
```

Comment:
All these `ll` should just be defined as `l := l.With`. Its in a nested scope so won't change the outer `l` and avoids bugs like the `l.Error` calls below and even some `l.Info` and `l.Debug` way below.